src/Pickles/Pickles.TestFrameworks/NUnit/NUnitExampleSignatureBuilder.cs

```diff
@@ -44,5 +48,8 @@ public Regex Build(ScenarioOutline scenarioOutline, string[] row)
 
             return new Regex(stringBuilder.ToString());
         }
+
+        private static readonly Regex punctuationCharactersRegex = new Regex(@"[\n\.-]+", RegexOptions.Compiled);
```

Comment:
Also, because the fields are static, I prefer the name to begin with an uppercase letter.